### PR TITLE
NanoID v3 compatability & numbers as letters filtering

### LINKED ISSUE #9: Upgrade to nanoid v3
Any chance this can be updated to support the latest version of nanoid? It appears the current dependency is listed as `"nanoid": "^2.0.0"` and in v3 a [few things have changed](https://github.com/ai/nanoid/releases/tag/3.0.0).

### PR DESCRIPTION
**Pull request by webdevian**
- Refactor to match the exports and API of NanoID v3 - Breaking Changes, Fixes #9 
- Match numbers that look like letters against naughty words (i.e. `A5S` matches `ass`)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ npm install nanoid-good
 
 ```js
 var en = require("nanoid-good/locale/en"); // you should add locale of your preferred language
-var nanoid = require("nanoid-good")(en);
+var nanoid = require("nanoid-good").nanonid(en);
```
Comment (y-gagar1n) [inline README.md:29]: typo?

Comment (webdevian) [reply README.md:29]: No, in v3 default export has been replaced with named export - https://github.com/ai/nanoid/releases/tag/3.0.0

Comment (y-gagar1n) [reply README.md:29]: I mean it should be `.nanoid`, not `.nanonid`

Comment (webdevian) [reply README.md:29]: Ah yes, you are correct

**Comment by y-gagar1n** [comment]
Thank you for your contribution! Looks ok, but I don't do frontend development anymore, so it might take some time to check it and publish new version. I'll do it this weekend.

File: README.md (hunk 1 above)
Comment (y-gagar1n) [reply README.md:29]: I see that you fixed the typo at line 60, but please fix this one too (line 29)

**Comment by y-gagar1n** [comment]
Everything looks good, please fix that one little typo and I'll accept.

**Comment by webdevian** [comment]
@y-gagar1n Fixed

**Comment by y-gagar1n** [comment]
Accepted and published a new version, thanks a lot!

**Pull request merged**